regexes/device/mobiles.yml

```diff
@@ -334,35 +337,24 @@ Nokia:
       model: '7'
     - regex: 'TA-10(04|12|52)'
       model: '8'
-    # 2018 Models
-    - regex: 'Nokia 8\.1'
-      model: '8.1'
     - regex: 'Nokia 8 Sirocco'
       model: '8 Sirocco'
-    - regex: 'Nokia 7\.1'
-      model: '7.1'
     - regex: 'Nokia 7 plus'
       model: '7 plus'
     - regex: 'Nokia 6\.1 Plus'
       model: '6.1 Plus'
-    - regex: 'Nokia 6\.1'
-      model: '6.1'
     - regex: 'Nokia 5\.1 Plus'
       model: '5.1 Plus'
-    - regex: 'Nokia 5\.1'
-      model: '5.1'
     - regex: 'Nokia 3\.1 Plus'
       model: '3.1 Plus'
-    - regex: 'Nokia 3\.1'
-      model: '3.1'
-    - regex: 'Nokia 2\.1'
-      model: '2.1'
     - regex: 'Nokia 1 Plus'
       model: '1 Plus'
-    - regex: 'Nokia 1'
-      model: '1'
```

Comment:
Were those detection incorrect? Couldn't see a simpler replacement 🤔

Reply:
these are redundant rules, and some rules such as `Nokia 1`
blocks the detect of models 1.3 and 1.4

https://github.com/matomo-org/device-detector/blob/9e931648250981b357d78c8183da99ce840a666a/regexes/device/mobiles.yml#L452-L453

Should I restore the changes back?

Reply:
Ah. I had searched for it, but guess I missed the other rule. No need to revert anything. It's fine to remove redundant rules.